cookbooks/fb_fstab/libraries/default.rb

```diff
@@ -205,26 +208,29 @@ def self.get_unmasked_base_mounts(format, node)
               data['mount_point'] == line_parts[1] &&
                 data['device'].start_with?('LABEL=')
             end
+
             raise e
           end
           # If someone has a more specific mount, don't use the original
           next if desired_mounts.any? do |_name, data|
             begin
```

Comment:
This changes a bit because rubocop chokes on this crazy mix of things, trying to do an implicit begin and then breaking things, so this is both a bit cleaner, more clear, and doesn't confuse rubocop